cluster/juju/layers/kubernetes-master/templates/kube-scheduler.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=Kubernetes Scheduler
 Documentation=http://kubernetes.io/docs/admin/kube-scheduler/
-After=unit.target kube-apiserver
-Requires=kube-apiserver
+After=unit.target kube-apiserver.service
```

Comment:
Do BOTH of these require apiserver? Or should the scheduler depend on the controller-manager?